src/BatchWriter/IPFS.ts

```diff
@@ -0,0 +1,36 @@
+import { inject, injectable } from 'inversify'
```

Comment:
I originally created the IPFS client as a layer in a module and then regretted doing so.

What do you think about refactoring this into a separate client (no DI) in Helpers/IPFSClient?

Reply:
Yep! Or maybe actually merge this one first and handle the refactor separate from the feature if you prefer

Reply:
Sure, either way, I think getting this merged soon is probably better time wise?

Reply:
Actually, both can be worked on at the same time, depending on what gets merged first will decide if we use the helper or not in this pr.